azure-pipelines.yml

```diff
@@ -107,7 +107,7 @@ jobs:
   - task: UsePythonVersion@0
     displayName: 'Use Python 3'
     inputs:
-      versionSpec: 3.8
+      versionSpec: 3.x
```

Comment:
@fengzhou-msft should we also roll back the CI task here?

Reply:
we can keep this to specify 3.8